lib/internal/cluster/master.js

```diff
@@ -72,6 +72,7 @@ cluster.setupMaster = function(options) {
 
   process.nextTick(setupSettingsNT, settings);
 
+  // XXX cluster masters turn debug on in all children? does that work?
```

Comment:
I believe there are tests for that. I distinctly remember having to deal with them :)